boards/arm/usb_kw24d512/Kconfig.defconfig

```diff
@@ -26,17 +26,10 @@ config MCG_VDIV0
 config MCG_FCRDIV
 	default 2
 
-if UART_MCUX
-
-config UART_MCUX_0
-	default y
-
-endif # UART_MCUX
-
 if PINMUX_MCUX
 
 config PINMUX_MCUX_PORTA
-	default y if UART_MCUX_0
+	default y if "$(dt_dev_exist_by_path,/soc/uart@4006a000)"
```

Comment:
Just for clarification, `dt_dev_exist_by_path` means it exists or it is enabled?

Reply:
Both.  Any suggestions on a better name to make this more clear?

`dt_dev_enabled_by_path`?